### ISSUE TITLE
Some of the macro names are out of date in the README

### ISSUE THREAD

**Issue by bryanhickerson** [kellyselden/ember-awesome-macros#68]
I was getting an import error for `lt` and looked at the addon folder and saw that `lt` was defined in `lt-key.js`. I noticed most of comparison macros are named similarly. 
```
Uncaught Error: Could not find module `ember-awesome-macros/lt` imported from `frontend/pods/components/appointments-list/component`
```

It's a bit more cryptic if you import in this way: `import { lt } from 'ember-awesome-macros';`, because you get the error:
```
Uncaught TypeError: (0 , _emberAwesomeMacros.lt) is not a function
```

**Comment by kellyselden** [comment]
Yeah I was trying to cram a few more macros before releasing another version. Want me to just release now?

**Comment by bryanhickerson** [comment]
It's fine with me if you want to hold off, I just wanted to raise it up for the sake of people trying out the addon.

**Comment by kellyselden** [comment]
Released v0.11.0. I'm still working out the best API in these pre 1.0 versions. Sorry for the breaking changes.
